Makefile

```diff
@@ -119,22 +122,23 @@ cluster-ip:
   CLUSTER_IP?=$(shell $(KUBECTL) get svc kubernetes -n default -o jsonpath="{.spec.clusterIP}")
 
 cluster-hub-ip-kind:
-  CLUSTER_IP_KIND?=$(shell $(KUBECTL) --kubeconfig=$(HUB_KIND_KUBECONFIG) config view | grep server | awk '{ print $2 }' | cut -f3 -d/ | cut -f1 -d:)
+  	CLUSTER_IP_KIND?=$(shell $(KUBECTL) --kubeconfig $(HOME)/hub-kubeconfig config view | grep server | awk '{ print $2 }' | cut -f3 -d/)
 
 bootstrap-secret: cluster-ip
 	cp $(KUBECONFIG) dev-kubeconfig
-	$(KUBECTL) config use-context $(KLUSTERLET_KUBECONFIG_CONTEXT)
+	$(KUBECTL) config use-context kind-$(MANAGED_CLUSTER)
```

Comment:
I think this will affect our e2e test.

Reply:
right, I see that now. How can I run the e2e test locally to debug ? `make test-e2e` ?

Reply:
yeah, you can run make test-e2e locally